src/cli_util.rs

```diff
@@ -1177,6 +1177,8 @@ pub fn short_operation_hash(operation_id: &OperationId) -> String {
 pub struct Args {
     #[command(flatten)]
     pub global_args: GlobalArgs,
+    #[command(flatten)]
+    pub early_args: EarlyArgs,
```

Comment:
It feels like this should be inside `GlobalArgs`. Would that work?

Reply:
i.e. `args.global_args.early_args`? Yes that works, I just thought this looked nicer :)

Reply:
The early args *are* global args, so I think it makes more sense there than to say that the CLI has early args and global args as separate things

Reply:
Okay I'll do that then. Neither option makes a big difference to the code structure since the args are pretty well-contained in `cli_util.rs`.

Reply:
As of this PR, it turns out it makes **no difference** at all because the `EarlyArgs` never get referenced outside of `handle_early_args()`.